src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,7 @@
+.catalog {
+  width: calc((4 * #{$cell-size}) + (#{$card-margin-left} * 3));
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin: 0 auto;
```

Comment:
<img width="902" alt="image" src="https://user-images.githubusercontent.com/102411130/226557557-8123070a-6514-4672-8562-3aeeaaa41bd9.png">

Reply:
Not fixed, try to remove top margin from card and this apacing for catalog, so that the catalog is indented from the rest of the BEM elements, and so that the cards are indented between themselves